test/test_ops.py

```diff
@@ -209,6 +210,7 @@ def to_cpu(arg):
     @unittest.skipIf(TEST_WITH_ASAN, "Skipped under ASAN")
     @onlyNativeDeviceTypes
     @ops(python_ref_db)
+    @skipIfTorchInductor("Takes too long for inductor")
```

Comment:
how do these tests get run ? I thought we were only invoking `test/test_modules.py` and `test/test_ops_gradients.py` ?

Reply:
You got me. I ran this one locally, and just decide to smuggle it here to avoid having local changes.